tests/settings.py

```diff
@@ -0,0 +1,29 @@
+HELPER_SETTINGS = {
+    "INSTALLED_APPS": [
+        "rules",
+        "djangocms_fil_permissions",
+        "djangocms_fil_permissions.test_utils.polls",
+    ],
+    "MIGRATION_MODULES": {
```

Comment:
do we need this if all the items are commented inside?

Reply:
Shouldn't be commented